Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -933,7 +933,41 @@ AXTextMarker AXTextMarker::findMarker(AXDirection direction, AXTextUnit textUnit
 
         return resultMarker;
     }
-    // FIXME: Implement the other AXTextUnits.
+
+    // Paragraphs must be handled differently from word + sentence boundaries, as there is no paragraph break iterator.
+    // Rather, paragraph boundaries are based on rendered newlines and changes in nodes/renderers.
```

Comment:
"changes in nodes/renderers" is a little bit vague. Is it possible to be more specific here — what changes from node-to-node or renderer-to-renderer are relevant?

Reply:
Will update this to be more specific!